ntbs-service/Models/NotificationBannerModel.cs

```diff
@@ -65,8 +66,7 @@ public NotificationBannerModel(Notification notification, bool showPadlock = fal
 
         private static string CalculateOutcome(Notification notification)
         {
-            return notification.TreatmentEvents
-                       .GetMostRecentTreatmentEvent()
+            return notification.TreatmentEvents.OrderForEpisodes().LastOrDefault()

```

Comment:
It might be slightly nicer to keep the `GetMostRecentTreatmentEvent` helper and change the definition so that it calls `OrderForEpisodes` - since this logic is now duplicated here and in the file above. Very minor thing though.

Reply:
Have made this change so we now use the `GetMostRecentTreatmentEvent` extension